symfony/plugins/orangehrmAdminPlugin/config/routes.yaml

```diff
@@ -110,6 +110,20 @@ apiv2_licenses:
   defaults:
     _api: OrangeHRM\Admin\Api\LicenseAPI
 
+apiv2_organization:
+  path: /api/v2/admin/organization/{id}
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET, PUT ]
+  defaults:
+    _api: OrangeHRM\Admin\Api\OrganizationAPI
+
+apiv2_organizations:
```

Comment:
Remove this entry,since we are not implementing `getAll`, `create`, `delete`

Reply:
removd